pkg/migrations/internal/seq_num.go

```diff
@@ -4,7 +4,7 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 172
+	CurrentDBVersionSeqNum = 175
```

Comment:
I assume you have a couple of migrations stacked up in review. just make sure they are pushed in order or update the sequence accordingly.